game/templates/game/board.html

```
@@ -303,6 +303,7 @@
                 <div class="card-title">Game Controls</div>
                 <div style="display: flex; flex-direction: column; gap: 8px;">
                     <button class="btn btn-gold" id="flipBtn">Flip Board</button>
+                    <button class="btn btn-gold" id="blindfoldBtn">Blindfold: OFF</button>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)